Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -150,7 +150,7 @@
     gloves: ClothingHandsGlovesCombat
     outerClothing: ClothingOuterHardsuitMedic
     shoes: ClothingShoesBootsMagSyndie
-    id: SyndiPDA
+    id: SyndiAgentPDA
```

Comment:
Wouldn't `SyndiMedicPDA` be a better name?

`SyndiAgentPDA` feels like it should be the name for the default version.
`SyndiPDA` would be the base, if needed.